adserver/templates/adserver/advertiser/overview.html

```diff
@@ -0,0 +1,101 @@
+{% extends "adserver/base.html" %}
+{% load crispy_forms_tags %}
+{% load humanize %}
+{% load i18n %}
+
+
+{% block title %}{% trans 'Advertiser Overview' %} - {{ advertiser }}{% endblock %}
+
+
+{% block breadcrumbs %}
+  {{ block.super }}
+  <li class="breadcrumb-item active">{{ advertiser }}</li>
+{% endblock breadcrumbs %}
+
+
+{% block content_container %}
+  <section>
+    <h1>{% blocktrans %}{{ advertiser }}{% endblocktrans %}</h1>
+    <p><small>Month to date overview for {{ advertiser }}.</small></p>
+
+    <div class="row row-cols-1 row-cols-md-4">
+      <div class="col mb-3">
+        <div class="card">
+          <div class="card-body">
+            <h3 class="card-title text-center">{{ report.total.ctr|floatformat:3 }}%</h3>
+            <h6 class="text-muted text-center">{% trans 'CTR' %}</h6>
+          </div>
+        </div>
+      </div>
+      <div class="col mb-3">
+        <div class="card">
+          <div class="card-body">
+            <h3 class="card-title text-center">{{ report.total.views|intcomma }}</h3>
+            <h6 class="text-muted text-center">{% trans 'Views' %}</h6>
+          </div>
+        </div>
+      </div>
+      <div class="col mb-3">
+        <div class="card">
+          <div class="card-body">
+            <h3 class="card-title text-center">{{ report.total.clicks|intcomma }}</h3>
+            <h6 class="text-muted text-center">{% trans 'Clicks' %}</h6>
+          </div>
+        </div>
+      </div>
+      <div class="col mb-3">
+        <div class="card">
+          <div class="card-body">
+            <h3 class="card-title text-center">${{ report.total.cost|floatformat:2|intcomma }}</h3>
+            <h6 class="text-muted text-center">{% trans 'Spend' %}</h6>
+          </div>
+        </div>
+      </div>
+    </div>
+    <p class="text-right mb-5"><a href="{% url 'advertiser_report' advertiser.slug %}?start_date={{ start_date|date:'Y-m-d' }}">{% trans 'detailed reporting' %} &raquo;</a></p>
```

Comment:
This might confuse some advertisers by being different than the default for this view (eg. 30 days vs. beginning of month).

We should probably make the filtered dates a bit more visible (especially since on my display it hides them)

![Screen Shot 2021-04-01 at 9 16 34 AM](https://user-images.githubusercontent.com/25510/113323576-f700b580-92ca-11eb-8fff-a65345a0a994.png)

Reply:
I was able to fix it by increasing the container it's in:

![Screen Shot 2021-04-01 at 9 23 37 AM](https://user-images.githubusercontent.com/25510/113324471-fd436180-92cb-11eb-9031-693cba398478.png)

Might be worth adjusting?

Reply:
This is a fair criticism but I'm probably going to handle it in a separate PR since it isn't directly related to the overview screen.

Reply:
I ended up fixing it in https://github.com/readthedocs/ethical-ad-server/pull/351